src/components/DivElement.js

```diff
@@ -1,6 +1,11 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
-
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super('', content);
+    this.tag = 'div';
```

Comment:
Remember to take advantage of the parent constructor. You can also assign 'div' to tag by passing it as the first parameter of super().